app/scrapers/__init__.py

```diff
@@ -23,6 +24,7 @@
     'e': Exalead(),
     'q': Quora(),
     't': Youtube()
+    'g': Gigablast()
```

Comment:
`g --> google`
if the results are coming than, that are all of google search and not gigablast